test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -210,15 +227,17 @@ verify_branches()
         echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
 
         cd "$1"
+
+        wt_bin="build/wt"
         for am in $3; do
-            echo "$1/wt verifying $2 access method $am..."
-            dir="$2/test/format/RUNDIR.$am"
-            WIREDTIGER_CONFIG="$EXT" ./wt $(bflag $1) -h "../$dir" verify table:wt
+            echo "$wt_bin verifying $2 access method $am..."
```

Comment:
The `$1` (i.e. the branch) is useful information to be dumped out here.

Reply:
Yep, good catch. I've updated it in aa3746878f698945f508158081bab167302af757